src/main/java/com/garbagemule/MobArena/ArenaListener.java

```diff
@@ -533,6 +540,9 @@ else if (monsters.removeMount(event.getEntity())) {
         else if (monsters.removeGolem(event.getEntity())) {
             arena.announce(Msg.GOLEM_DIED);
         }
+        else if (event.getEntity() instanceof Projectile) {
+            arena.getProjectileManager().removeProjectile((Projectile) event.getEntity());
+        }
```

Comment:
Same deal here with sanity checks. Don't assume that the projectile was shot from a player in _this_ arena.